ui/perfherder/compare/RetriggerModal.jsx

```diff
@@ -0,0 +1,223 @@
+      } else if (!currentRetriggerRow.originalRetriggerableJobId) {
+        inputTitle = 'Disabled input because there are no jobs to retrigger';
+      }
+    } else if (!currentRetriggerRow.newRetriggerableJobId) {
+      inputTitle = 'Disabled input because there are no jobs to retrigger';
+    }
+
+    return inputTitle;
+  };
+
+  isValueValid = value => {
+    const { maxRetriggersValue } = this.props;
+    const regex = /^\d+$/;
+
+    if (!value.match(regex)) {
+      return false;
+    }
+    const parsedValue = parseInt(value, 10);
+    return parsedValue >= 0 && parsedValue <= maxRetriggersValue;
+  };
+
+  handleChange = event => {
+    const inputName = event.target.name;
+    const updates = {
+      [inputName]: 0,
+      invalidInput: false,
+    };
+
+    if (this.isValueValid(event.target.value)) {
+      updates[inputName] = event.target.value;
+    } else {
+      updates.invalidInput = true;
+    }
+
+    this.setState(updates);
+  };
+
+  onRetriggerClick = event => {
+    const { updateAndClose } = this.props;
+    const { baseRetriggerTimes, newRetriggerTimes } = this.state;
+
+    updateAndClose(event, {
+      baseRetriggerTimes: Number.isInteger(baseRetriggerTimes)
+        ? baseRetriggerTimes
+        : parseInt(baseRetriggerTimes, 10),
+      newRetriggerTimes: Number.isInteger(newRetriggerTimes)
+        ? newRetriggerTimes
+        : parseInt(newRetriggerTimes, 10),
+    });
+  };
+
+  render() {
+    const {
+      showModal,
+      toggle,
+      isBaseAggregate,
+      maxRetriggersValue,
+      currentRetriggerRow,
+    } = this.props;
+    const { invalidInput } = this.state;
+
+    return (
+      <Modal
+        isOpen={showModal}
+        onOpened={this.onOpened}
+        onClosed={this.onClosed}
+      >
+        <ModalHeader toggle={toggle}>Retrigger Jobs</ModalHeader>
+        <Form>
+          <ModalBody>
+            <div className="row">
+              <FormGroup
+                className="col-xs-10 col-sm-6 col-md-6 col-lg-6 form-inline"
+                inline
+              >
+                <InputGroup title={this.getInputTitle(true)}>
+                  <InputGroupAddon addonType="prepend">
+                    <InputGroupText>Base revision:</InputGroupText>
+                  </InputGroupAddon>
+                  <Input
+                    data-testid="input baseRetriggerTimes"
+                    defaultValue={this.getInitialValue(true)}
+                    min={0}
+                    max={maxRetriggersValue}
+                    onChange={this.handleChange}
+                    name="baseRetriggerTimes"
+                    type="number"
+                    disabled={
+                      isBaseAggregate ||
+                      !currentRetriggerRow.originalRetriggerableJobId
+                    }
+                  />
+                </InputGroup>
+              </FormGroup>
+              <FormGroup
+                className="col-xs-10 col-sm-6 col-md-6 col-lg-6 form-inline"
```

Comment:
Why not use a`Col` component instead of making this FormGroup kinda act like a column?

Reply:
I change it to Col